Megaparsec/Errors/Bundle.lean

```diff
@@ -27,7 +27,7 @@ private def makePEBfs [Printable β] [ToString E] [Streamable ℘] : ((String 
           | .trivial _     none _ => 1
           | .trivial _ (some x) _ => errorItemLength x
           | .fancy   _         xs =>
-            xs.foldl (fun acc e => max acc (errorFancyLength e)) 1
+            xs.fold (fun acc e _ => max acc (errorFancyLength e)) 1
```

Comment:
What's the difference between what we had before and this? I didn't think about it too much, I admit, but I'd like to know why this change is necessary.

Reply:
`RBMap` doesn't have `foldl`, and `xs` was changed from `List` to `RBMap`.

Reply:
🤦 thanks for explaining! I should have caught it during the review.